app/components/Map.js

```diff
@@ -8,7 +8,7 @@ export default class Map extends Component {
   }
   render() {
     return (
-     <View style={styles.container}>
+     <View style={[styles.container, { backgroundColor: 'transparent'}]}>
```

Comment:
note: we'll need to see if hiding is possible with native's style. if not, we will need to use React-router